static/elements/chromedash-feature-page.js

```diff
@@ -0,0 +1,264 @@
+
+export class ChromedashFeaturePage extends LitElement {
+  static get styles() {
+    return [
+      ...SHARED_STYLES,
+      css`
+        #feature {
+          background: var(--card-background);
+          border-radius: var(--default-border-radius);
+          border: var(--card-border);
+          box-shadow: var(--card-box-shadow);
+
+          box-sizing: border-box;
+          word-wrap: break-word;
+          margin-bottom: var(--content-padding);
+          max-width: var(--max-content-width);
+        }
+        #feature ul {
+          list-style-position: inside;
+          list-style: none;
+        }
+        section {
+          margin-bottom: 1em;
+        }
+        section h3 {
+          margin: 24px 0 12px;
+        }
+        section label {
+          font-weight: 500;
+          margin-right: 5px;
+        }
+        
+        #consensus li {
+          display: flex;
+        }
+        #consensus li label {
+          width: 125px;
+        }
+        
+        @media only screen and (max-width: 700px) {
+          #feature {
+            border-radius: 0 !important;
+            margin: 7px initial !important;
+          }
+        }
+        
+        @media only screen and (min-width: 701px) {
+          #feature {
+            padding: 30px 40px;
+          }
+        }
+    `];
+  }
+
+  static get properties() {
+    return {
+      feature: {type: Object},
+      featureId: {type: Number},
+      loading: {attribute: false},
+    };
+  }
+
+  constructor() {
+    super();
+    this.feature = {};
+    this.featureId = 0;
+    this.loading = true;
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.fetchFeature();
+  }
+
+  fetchFeature() {
+    this.loading = true;
+    window.csClient.getFeature(this.featureId).then(
+      (feature) => {
+        this.loading = false;
+        this.feature = feature;
+      }).catch(() => {
+      const toastEl = document.querySelector('chromedash-toast');
+      toastEl.showMessage('Some errors occurred. Please refresh the page or try again later.');
+    });
+  }
+
+  renderFeatureContent() {
+    return html`
+      ${this.feature.unlisted ? html`
+        <section id="access">
+        <p><b>This feature is only shown in the feature list to users with
+        edit access.</b></p>
+        </section>
+      `: nothing}
+   
+      ${this.feature.summary ? html`
+        <section id="summary">
+          <p class="preformatted">${autolink(this.feature.summary)}</p>
+        </section>
+      `: nothing}
+
+      ${this.feature.motivation ? html`
+        <section id="motivation">
+          <h3>Motivation</h3>
+          <p class="preformatted">${autolink(this.feature.motivation)}</p>
+        </section>
+      `: nothing}
+
+      ${this.feature.resources && this.feature.resources.samples ? html`
+        <section id="demo">
+          <h3>${this.feature.resources.length == 1 ? 'Demo' : 'Demos'}</h3>
```

Comment:
`resources` is an object, I think you need `resources.samples` which is either null or an array.

Reply:
Fixed!